### PR TITLE
Tutorial Page

### LINKED ISSUE #197: Tutorial
**Is your feature request related to a problem? Please describe.**
We need to implement tutorial page for new visitors

**Describe the solution you'd like**
I would like to implement simple tutorials for them.

**Describe alternatives you've considered**
x

**Additional context**
x

### PR DESCRIPTION
**Pull request by kosumin**
Related Issue: #197 
it resolves #197

# Major changes

1. Implement Tutorial Page

image

image

image

image

image

image

# Minor changes

1. Make links on side bar/ header disabled when user is in init condition(during sign up -> tutorial -> init)
2. Change the default tab on collecton detail page when clicking reply button on Collection Card

image

### Checklist
This pull request...
- [x] has preceding issues related with it (if resolves some issues, metions them with 'resolve')
- [x] doesn't include too many changes (you should focus on the specific feature in a PR)
- [x] elaborately explains what feature is added, which part is fixed or improved
- [x] has step by step instructions for collaborators, if needed
- [x] has attached screenshots, if appropriate

My codes...
- [x] include enough tests for added and changed parts
- [x] pass all the new and existing tests
- [x] include appropriate comments
- [x] don't introduce unnecessary warnings

### CODE REVIEW

File: frontend/src/containers/User/AccountSetting/AccountSetting.js [hunk 1]
```diff
@@ -145,7 +145,7 @@ class AccountSetting extends Component {
                     <input
                       id="editUserName"
                       type="text"
-                      value={this.state.username}
+                      value={this.state.username || ""}
```
Comment (davin111) [inline frontend/src/containers/User/AccountSetting/AccountSetting.js]: What is meaning of this part?

Comment (kosumin) [reply frontend/src/containers/User/AccountSetting/AccountSetting.js]: When I didn't add this part, test warning occurred. (A component is changing a controlled input of type text to be uncontrolled.)

Comment (kosumin) [reply frontend/src/containers/User/AccountSetting/AccountSetting.js]: So I follow the solution from here: https://stackoverflow.com/questions/47012169/a-component-is-changing-an-uncontrolled-input-of-type-text-to-be-controlled-erro

Comment (davin111) [reply frontend/src/containers/User/AccountSetting/AccountSetting.js]: Ok. When I saw those errors, mostly it is due to ‘undefined’ values to inputs.

Comment (kosumin) [reply frontend/src/containers/User/AccountSetting/AccountSetting.js]: Oh then is it okay to leave the values like this? Or could you suggest a better way?

Comment (davin111) [reply frontend/src/containers/User/AccountSetting/AccountSetting.js]: I think it’s ok if your solution works. I just wanted to let you know potential reasons. Maybe you can find the time where you unintentionally assign inappropriate(undefined) values.

**Pull request merged**